src/services/quotes/quote-sources/dodo-quote-source.ts

```diff
@@ -0,0 +1,93 @@
+import { Chains } from '@chains';
+import { AlwaysValidConfigAndContextSource } from './base/always-valid-source';
+import { QuoteParams, QuoteSourceMetadata, SourceQuoteResponse } from './types';
+import qs from 'qs';
+import { addQuoteSlippage, calculateAllowanceTarget, failed } from './utils';
+import { parseUnits } from 'viem';
+
+const DODO_DEX_METADATA: QuoteSourceMetadata<DodoDexSupport> = {
+  name: 'DODO Dex',
```

Comment:
I think we can go with DODO
```suggestion
  name: 'DODO',
```